ldap_auth_provider.py

```diff
@@ -62,18 +49,11 @@ class LDAPMode(object):
 
 
 class LdapAuthProvider(object):
-    __version__ = "0.1"
```

Comment:
we already have a module-level __version__, so let's bin this.

Reply:
I really appreciation how they're different too. 😄